auth-web/src/store/modules/business.ts

```diff
@@ -37,75 +32,98 @@ export default class BusinessModule extends VuexModule {
 
   @Mutation
   public setBusinesses (businesses: Business[]) {
-    this.businesses = businesses
+    this.businesses = [...businesses]
   }
 
-  @Action({ commit: 'setBusinesses', rawError: true })
-  public async syncBusinesses (): Promise<Business[]> {
-    const organization = this.context.rootState.org.currentOrganization
+  @Action({ rawError: true })
+  public async syncBusinesses (): Promise<void> {
+    /** Returns NR's approved name. */
+    const getApprovedName = (nr): string =>
+      nr.names.find(name => [NrState.APPROVED, NrState.CONDITION].includes(name.state))?.name
+
+    /** Returns True if NR is approved for incorporation. */
+    const isApprovedForIa = (nr): boolean =>
+      (nr.state === NrState.APPROVED &&
+        nr.actions.some(action => action.filingName === LearFilingTypes.INCORPORATION))
+
+    /** Returns True if NR is conditionally approved. */
+    const isConditionallyApproved = (nr): boolean =>
+      (nr.state === NrState.CONDITIONAL &&
+        [NrConditionalStates.RECEIVED, NrConditionalStates.WAIVED].includes(nr.consentFlag))
+
+    /** Returns True if NR is approved for one stop registration. */
+    const isApprovedForRegistration = (nr): boolean =>
+      nr.actions.some(action => action.filingName === LearFilingTypes.REGISTRATION)
+
+    // initialize store
+    this.setBusinesses([])
```

Comment:
This clears the old table (which may be visible through the semi-transparent loading spinner.